roles/common/tasks/main.yml

```diff
@@ -73,11 +73,23 @@
 - include: hwraid.yml
   when: common.hwraid.enabled
 
+
+- name: Fetch Ursula revision
+  local_action: command git describe --tags
+  run_once: true
+  register: ursula_git_describe
```

Comment:
This needs to not run as sudo, since most everything is set to run as sudo in jenkins and other places, this task needs a ```sudo: no```